packages/tools/addon/helpers/cs-field-editor-for.js

```diff
@@ -16,4 +25,9 @@ export function csFieldEditorFor([content, fieldName], { variant }) {
   return `${prefix}field-editors/${type}-editor`;
 }
 
-export default Ember.Helper.helper(csFieldEditorFor);
+export default helper(csFieldEditorFor);
+
+function fieldOptions(content, fieldName) {
```

Comment:
I might get rid of this in favor of ensuring `meta.options` defaults to an empty has when using `metaForField`